doc/api/test.md

```diff
@@ -515,6 +515,113 @@ test('spies on an object method', (t) => {
 });
 ```
 
+### Timers
+
+Mocking timers is a technique commonly used in software testing to simulate and
+control the behavior of timers, such as `setInterval` and `setTimeout`,
+without actually waiting for the specified time intervals.
+
+Look at [`MockTimers`][] class to check out all methods
+and supported features from this API.
+
+This allows developers to write more reliable and
+predictable tests for time-dependent functionality.
+
+The example below shows how to mock `setTimeout`.
+Using `.enable(['setTimeout']);`
+it'll mock the `setTimeout` from both [node:timers](./timers.md),
+[node:timers/promises](./timers.md#timers-promises-api) modules,
+and from the Node.js global context.
```

Comment:
```suggestion
it will mock the `setTimeout` functions in the [node:timers](./timers.md) and
[node:timers/promises](./timers.md#timers-promises-api) modules,
as well as from the Node.js global context.
```